vendor/github.com/vmware/go-nfs-client/nfs/rpc/client.go

```diff
@@ -135,11 +142,24 @@ func (c *Client) Call(call interface{}) (io.ReadSeeker, error) {
 		case Success:
 			return res, nil
 		case ProgUnavail:
-			return nil, fmt.Errorf("PROG_UNAVAIL")
+			return nil, fmt.Errorf("rpc: PROG_UNAVAIL - server does not recognize the program number")
 		case ProgMismatch:
-			return nil, fmt.Errorf("rpc: PROG_MISMATCH")
+			return nil, fmt.Errorf("rpc: PROG_MISMATCH - program version does not exist on the")
```

Comment:
seems like there might need a bit more on the end here. e.g. `program version does not exist on the server`?